woocommerce/woocommerce-gateway-stripe
# feat: use payment request settings account keys

# Changes proposed in this Pull Request:

Almost forgot about this little thing :D 
We're using some mocked data for the "Payment request" button.
Let's use the store's account keys.
![Screen Shot 2021-10-08 at 4 22 43 PM](https://user-images.githubusercontent.com/273592/136626181-6f7568aa-ea40-4c59-88a5-6e28e9ade0ba.png)


(when loaded with HTTP, instead of HTTPS:)

![Screen Shot 2021-10-08 at 4 33 05 PM](https://user-images.githubusercontent.com/273592/136627099-0ab3897d-fc24-462a-8d09-2ead2ee69dce.png)

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Ensure your account is connected
- Start your Jurasic Tube tunnel
- Ensure you're using HTTPS to access your site
- Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&area=payment_requests`
- Button should now appear!
